src/ui/public/chrome/api/angular.js

```diff
@@ -56,6 +56,13 @@ module.exports = function (chrome, internals) {
           return chrome;
         }
       };
+    })
+    .run(function ($rootScope, $cookies) {
+      $rootScope.$watch(function () {
+        return $cookies.sid;
```

Comment:
Does this value change? If not, it'd be great if we could move it to the chrome. That way, apps not based on angular can access this useful tidbit.